### PR TITLE
A post to officially announce the Development Fund.

### PR DESCRIPTION
**Pull request by coppolaemilio**
The post is ready for review, but I'm still trying to figure out a good image to show for the article.
I also did some further updates on the `/donate` page to make it more focused on one-time donations.

### CODE REVIEW

File: collections/_article/godot-developer-fund.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us keep the project alive.
+ 
+
+## Reducing Platform Fees, Amplifying Impact
```
Comment (JohnVeness) [inline collections/_article/godot-developer-fund.md]: I think we normally use "Sentence case" for headings, not "Title Case".
```suggestion
## Reducing platform fees, amplifying impact
```

File: collections/_article/godot-developer-fund.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The Development Fund will make The Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
+
+If you are currently supporting us [through Patreon](https://www.patreon.com/godotengine), we encourage you to switch to the [Godot Development Fund](https://fund.godotengine.org) to increase the effectiveness of your donations.
+
+## Giving Back to the Community
```
Comment (JohnVeness) [inline collections/_article/godot-developer-fund.md]:
```suggestion
## Giving back to the community
```

File: collections/_article/godot-developer-fund.md (hunk 1 above)
Comment (coppolaemilio) [reply collections/_article/godot-developer-fund.md]: That is correct! Thanks!

File: collections/_article/godot-developer-fund.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/).With you, we can achieve even more!
```
Comment (JohnVeness) [inline collections/_article/godot-developer-fund.md]:
```suggestion
The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/). With you, we can achieve even more!
```

File: collections/_article/godot-developer-fund.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Every donation, no matter the size, helps us immensely! With your help we can hire more developers, create high quality demos, pay for hosting services and infrastructure, and make the best possible Godot Engine. The more funds we have, the more we can give back to you and the entire gamedev community.
+
+## Future Enhancements
```
Comment (JohnVeness) [inline collections/_article/godot-developer-fund.md]:
```suggestion
## Future enhancements
```

File: collections/_article/godot-developer-fund.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+While this is a good, fully functioning starting point for the platform, we will keep improving the Developer Fund.
+
+Moving forward, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 
+
+## Freedom and independence
+
+By relying more on user-funded contributions, we reduce our dependence on large grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open-source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.
+
+
+## A Heartfelt Thanks
```
Comment (JohnVeness) [inline collections/_article/godot-developer-fund.md]:
```suggestion
## A heartfelt thanks
```

File: collections/_article/godot-developer-fund.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing the new Godot Development Fund"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us keep the project alive.
```
Comment (clayjohn) [inline collections/_article/godot-developer-fund.md]:
```suggestion
We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
```

I'm not sure my suggestion is perfect. But I do think we should use more subtle language than "help us keep the project alive". While technically it has the same meaning as my suggestion, it also makes it sound like the project is out of funds (which was a common misconception from our last funding post). 

Personally, I think we should be highlighting 2 things:
1. Longevity (we need to ensure that development pace is maintained as long as possible)
2. Independence. The more funding we have from individuals, the less risk there is that Godot is "selling out". This is more of a public perception thing than an actual problem.

File: collections/_article/godot-developer-fund.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/). With you, we can achieve even more!
+
+Every donation, no matter the size, helps us immensely! With your help we can hire more developers, create high quality demos, pay for hosting services and infrastructure, and make the best possible Godot Engine. The more funds we have, the more we can give back to you and the entire gamedev community.
+
+## Future enhancements
+
+While this is a good, fully functioning starting point for the platform, we will keep improving the Developer Fund.
+
+Moving forward, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms.
```
Comment (clayjohn) [inline collections/_article/godot-developer-fund.md]:
```suggestion
In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 
```

File: collections/_article/godot-developer-fund.md (hunk 6 above)
Comment (clayjohn) [reply collections/_article/godot-developer-fund.md]: Ah, I see you do that more below.

**Review by clayjohn — COMMENTED** [review]
Great work! 

I'm not in front of a computer, so I don't have a suggestion for now. But I think we need to tighten up the independence section a little bit. Right now it comes off a little bit like we don't want corporate donations, which isn't true.

File: collections/_article/godot-developer-fund.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Introducing the new Godot Development Fund"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]:
```suggestion
We are excited to introduce a major improvement in funding the development efforts of the Godot Engine – the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
```

File: collections/_article/godot-developer-fund.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+ 
+
+## Reducing platform fees, amplifying impact
+
+The Development Fund will make The Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]:
```suggestion
The Development Fund will make the Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
```

Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]: Should we clarify what platform fees mean here? It's not just about cutting the middleman, Patreon is a platform for commercial subscriptions and thus they charge VAT. So it's not just their 5-8% cut + 5% payment processor, but an additional +20-25% VAT for EU people.

File: collections/_article/godot-developer-fund.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/). With you, we can achieve even more!
+
+Every donation, no matter the size, helps us immensely! With your help we can hire more developers, create high quality demos, pay for hosting services and infrastructure, and make the best possible Godot Engine. The more funds we have, the more we can give back to you and the entire gamedev community.
+
+## Future enhancements
+
+While this is a good, fully functioning starting point for the platform, we will keep improving the Developer Fund.
+
+In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms.
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]:
```suggestion
In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms.
```

File: collections/_article/godot-developer-fund.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+## Future enhancements
+
+While this is a good, fully functioning starting point for the platform, we will keep improving the Developer Fund.
+
+In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 
+
+## Freedom and independence
+
+By relying more on user-funded contributions, we reduce our dependence on large grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open-source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]:
```suggestion
By relying more on recurring user-funded contributions and company sponsorships, we reduce our dependence on large one-time grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.
```

Maybe helps address @clayjohn's concern? We do need to encourage for corporate sponsorships _a lot_ more, which are usually upheld for multiple years and help us plan ahead.

Comment (QbieShay) [inline collections/_article/godot-developer-fund.md]: you, the users -> you and the rest of the Godot community

(perhaps?)

File: collections/_article/godot-developer-fund.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+title: "Introducing the new Godot Development Fund"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
+
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md:11]:
```suggestion
 
 The Development Fund was [soft-launched in July](/article/funding-breakdown-and-hiring-process/) and already welcome a lot of new and old sponsors, but we need to take it to the next level to sustain the current development pace, and increase it with new hires.
```

File: collections/_article/godot-developer-fund.md (hunk 11 above)
Comment (coppolaemilio) [reply collections/_article/godot-developer-fund.md]: yeah, I like this edit

File: collections/_article/godot-developer-fund.md (hunk 9 above)
Comment (coppolaemilio) [reply collections/_article/godot-developer-fund.md]: I didn't have the numbers exactly about how much Patreon got, so I wasn't sure about adding hard numbers. If those are the percentages we could add it 100%

Comment (akien-mga) [reply collections/_article/godot-developer-fund.md]: Well I wouldn't try to give clear numbers either, but I think detailing that we're saving on both platform fees and, where applicable, VAT, would be useful.

File: collections/_article/godot-developer-fund.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Emi
+image: /storage/blog/covers/developer-fund.jpg
+date: 2023-09-12 11:51:10
+---
+
+We are excited to introduce a major improvement in funding the development efforts of the Godot Engine – the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
+ 
+ The Development Fund was [soft-launched in July](/article/funding-breakdown-and-hiring-process/) and already welcome a lot of new and old sponsors, but we need to take it to the next level to sustain the current development pace, and increase it with new hires.
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]:
```suggestion
The Development Fund was [soft-launched in July](/article/funding-breakdown-and-hiring-process/) and already welcomed a lot of new and old sponsors, but we need to take it to the next level to sustain the current development pace, and increase it with new hires.
```

File: collections/_article/godot-developer-fund.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+## Reducing platform fees, amplifying impact
+
+The Development Fund will make the Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
```
Comment (akien-mga) [inline collections/_article/godot-developer-fund.md]:
```suggestion
The Development Fund will make the Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, and as a non-profit we do not charge VAT on donations, unlike a platform like Patreon does. So your contributions go directly towards the development and improvement of the Godot Engine itself.
```

Not super happy with the phrasing, but I think it's worth mentioning.

**Review by akien-mga — APPROVED** [review]
Looks good to me!

(Remember to squash on merge ;))

**Pull request merged**